ui/src/Assignments/AssignmentCard.tsx

```diff
@@ -174,7 +174,8 @@ function AssignmentCard({
                 ref={assignmentEditRef}
                 className="personRoleColor"
                 data-testid={`editPersonIconContainer-${assignment.id}`}
-                onClick={toggleEditMenu}>
+                onClick={toggleEditMenu}
+                onKeyDown={toggleEditMenu}>
```

Comment:
Does this mean any key will open the menu?

Reply:
If so, I'm rejecting till we make it a specific key. Seems to happen in other places too.